src/components/organisms/LandingContent/LandingContent.tsx

```diff
@@ -25,7 +25,9 @@ const LandingContent: FC<LandingContentProps> = ({ className }) => {
           <Button
             appearance={AppearanceTypes.Secondary}
             children={t('button.check_tutorial')}
+            href={isUserLoggedIn ? '/manual' : undefined}
```

Comment:
**[non-blocker]**

Not too important, but the condition here is unnecessary. Can be just href="/manual" since we won't show the button if it's not manual anyway (if user is not logged in)